app/assets/stylesheets/views/notifications.scss

```
@@ -0,0 +1,148 @@
+@import '../config/import';
+
+.notification {
+  --unseen-border-width: 0px;
+  --padding: var(--su-3);
+  padding: var(--padding);
+  padding-left: calc(var(--padding) - var(--unseen-border-width));
+
+  display: flex;
+  color: var(--base-80);
+  border-radius: 0;
+
+  @media (min-width: $breakpoint-s) {
+    --padding: var(--su-4);
+  }
+  @media (min-width: $breakpoint-m) {
+    --padding: var(--su-6);
```

When I first saw this page, I felt that the padding looked very generous, and I think it has to do with the fact that other surfaces in the app are tighter in their spacing. What if we tightened this up by reducing the size here?

```suggestion
    --padding: var(--su-5);
```

`var(--su-4)` seemed a bit too tighter because of the `crayons-avatar--xl` we're using, so that's how why I think `var(--su-5)` would be more appropriate.